recipe/meta.yaml

```diff
@@ -8,28 +8,40 @@ package:
 source:
   url: http://www.openfst.org/twiki/pub/GRM/NGramDownload/{{ name }}-{{ version }}.tar.gz
   sha256: 8af30f0f2e823ccd7b856093a0b395cd481c599884b76a6361e8b37812d89d87
+  patches:
+    - patches/0001-Add-CMake-files.patch  # [win]
 
 build:
-  number: 1
-  skip: true  # [win]
+  number: 2
 
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - make
-    - gnuconfig   # [unix]
+    - make                    # [unix]
+    - gnuconfig               # [unix]
+    - cmake                   # [win]
+    - ninja                   # [win]
+    - m2-patch                # [win]
   host:
     - openfst ==1.8.2
+    - dlfcn-win32             # [win]
   run:
     - openfst ==1.8.2
+    - ucrt                    # [win]
+
+
+{% set ngram_binaries = [
```

Comment:
Do you want to just test all of them?

Reply:
Oh yes, that's just an oversight on my part!